2021-08-28 00:29:25,407 INFO    MainThread:30955 [wandb_setup.py:_flush():69] setting env: {'entity': 'rishanthrajendhran', 'project': 'eigengame', 'sweep_id': 't2dgom1t', 'root_dir': '/Users/rishanthrajendhran/Desktop/EigenGame', 'run_id': 'knz0s54w', 'sweep_param_path': '/Users/rishanthrajendhran/Desktop/EigenGame/wandb/sweep-t2dgom1t/config-knz0s54w.yaml'}
2021-08-28 00:29:25,407 INFO    MainThread:30955 [wandb_setup.py:_flush():69] setting login settings: {}
2021-08-28 00:29:25,407 INFO    MainThread:30955 [wandb_init.py:_log_setup():336] Logging user logs to /Users/rishanthrajendhran/Desktop/EigenGame/wandb/run-20210828_002925-knz0s54w/logs/debug.log
2021-08-28 00:29:25,407 INFO    MainThread:30955 [wandb_init.py:_log_setup():337] Logging internal logs to /Users/rishanthrajendhran/Desktop/EigenGame/wandb/run-20210828_002925-knz0s54w/logs/debug-internal.log
2021-08-28 00:29:25,408 INFO    MainThread:30955 [wandb_init.py:init():369] calling init triggers
2021-08-28 00:29:25,408 INFO    MainThread:30955 [wandb_init.py:init():376] wandb.init called with sweep_config: {'learningRate': 5e-07}
config: {'xDim': (100000, 4), 'numIterations': 10000, 'k': 4, 'learningRate': 1, 'type': 'modified'}
2021-08-28 00:29:25,408 INFO    MainThread:30955 [wandb_init.py:init():418] starting backend
2021-08-28 00:29:25,408 INFO    MainThread:30955 [backend.py:_multiprocessing_setup():72] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-08-28 00:29:25,426 INFO    MainThread:30955 [backend.py:ensure_launched():132] starting backend process...
2021-08-28 00:29:25,435 INFO    MainThread:30955 [backend.py:ensure_launched():137] started backend process with pid: 30967
2021-08-28 00:29:25,444 INFO    MainThread:30955 [wandb_init.py:init():423] backend started and connected
2021-08-28 00:29:25,448 INFO    MainThread:30955 [wandb_run.py:_config_callback():787] config_cb None None {'learningRate': 5e-07}
2021-08-28 00:29:25,449 INFO    MainThread:30955 [wandb_init.py:init():465] updated telemetry
2021-08-28 00:29:25,450 INFO    MainThread:30955 [wandb_init.py:init():484] communicating current version
2021-08-28 00:29:26,370 INFO    MainThread:30955 [wandb_init.py:init():489] got version response upgrade_message: "wandb version 0.12.1 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-08-28 00:29:26,370 INFO    MainThread:30955 [wandb_init.py:init():497] communicating run to backend with 30 second timeout
2021-08-28 00:29:26,735 INFO    MainThread:30955 [wandb_init.py:init():522] starting run threads in backend
2021-08-28 00:29:31,742 INFO    MainThread:30955 [wandb_run.py:_console_start():1565] atexit reg
2021-08-28 00:29:31,743 INFO    MainThread:30955 [wandb_run.py:_redirect():1412] redirect: SettingsConsole.REDIRECT
2021-08-28 00:29:31,743 INFO    MainThread:30955 [wandb_run.py:_redirect():1417] Redirecting console.
2021-08-28 00:29:31,745 INFO    MainThread:30955 [wandb_run.py:_redirect():1479] Redirects installed.
2021-08-28 00:29:31,745 INFO    MainThread:30955 [wandb_init.py:init():546] run started, returning control to user process
2021-08-28 00:29:34,566 INFO    MainThread:30955 [wandb_run.py:_atexit_cleanup():1535] got exitcode: 1
2021-08-28 00:29:34,567 INFO    MainThread:30955 [wandb_run.py:_restore():1507] restore
2021-08-28 00:29:39,719 INFO    MainThread:30955 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
}
pusher_stats {
}

2021-08-28 00:29:40,178 INFO    MainThread:30955 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  total_bytes: 789
}

2021-08-28 00:29:40,579 INFO    MainThread:30955 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 789
  total_bytes: 6017
}

2021-08-28 00:29:41,211 INFO    MainThread:30955 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15375
  total_bytes: 15375
}

2021-08-28 00:29:41,317 INFO    MainThread:30955 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15375
  total_bytes: 15375
}

2021-08-28 00:29:41,423 INFO    MainThread:30955 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15375
  total_bytes: 15375
}

2021-08-28 00:29:41,530 INFO    MainThread:30955 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15375
  total_bytes: 15375
}

2021-08-28 00:29:41,635 INFO    MainThread:30955 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15375
  total_bytes: 15375
}

2021-08-28 00:29:41,738 INFO    MainThread:30955 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15375
  total_bytes: 15375
}

2021-08-28 00:29:41,841 INFO    MainThread:30955 [wandb_run.py:_wait_for_finish():1657] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15375
  total_bytes: 15375
}

2021-08-28 00:29:43,248 INFO    MainThread:30955 [wandb_run.py:_show_files():1879] logging synced files
